Sources/VLCFirstStepsBaseViewController.m

```diff
@@ -59,6 +71,52 @@ - (void)updateTheme
 }
 
 - (void)setupPage
+{
+    [self configurePage];
+    self.pageTitleContainer.hidden = self.isCompactHeight;
```

Comment:
hide `pageTitleContainer` for compact height - it will be readable as `title` in `navigationBar`